src/stirling/source_connectors/socket_tracer/protocols/cql/parse_test.cc

```diff
@@ -103,8 +103,8 @@ TEST_F(CQLParserTest, BadOpcode) {
 }
 
 TEST_F(CQLParserTest, LengthTooLarge) {
-  // Length is 0x01000000
-  constexpr uint8_t kBadLengthFrame[] = {0x04, 0x00, 0x00, 0x06, 0xff, 0x01,
+  // Length is 0x020000000
+  constexpr uint8_t kBadLengthFrame[] = {0x04, 0x00, 0x00, 0x06, 0x07, 0x20,
                                          0x00, 0x00, 0x00, 0x00, 0x00};
```

Comment:
See the [CQL spec](https://github.com/apache/cassandra/blob/4d2ee09ffabf7fb086fb9e2bd042fc946e72c3ec/doc/native_protocol_v4.spec#L68-L73) for how this should be laid out. Bytes 6-9 will be be a 32 bit signed integer representing the length.

Previously the 5th byte (0xff) was set to an [invalid opcode](https://github.com/pixie-io/pixie/blob/401c92c1ed086eaa35dbe17b4fbcfd96e7b1ca4f/src/stirling/source_connectors/socket_tracer/protocols/cql/types.h#L41-L58), so [this logic](https://github.com/pixie-io/pixie/blob/401c92c1ed086eaa35dbe17b4fbcfd96e7b1ca4f/src/stirling/source_connectors/socket_tracer/protocols/cql/parse.cc#L51) was triggering instead.